### DISCUSSION TITLE
Welcome to Sco-GEM Discussions!

### DISCUSSION THREAD

**Discussion by sulheim** [SysBioChalmers/Sco-GEM#123] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about the Sco-GEM model and the associated scripts.
  * Share ideas for model improvement and development.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by naziakhatoon-bit** [comment]
what biological features are enabled in your GEM to adapt complex media composition?
